src/turbnba.F

```diff
@@ -230,8 +226,8 @@ subroutine turbnba(nstep,uh,ruh,uf,ruf,vh,rvh,vf,rvf,mh,rmh,mf,rmf,zf,c1,c2,rho,
       enddo
 
 #ifdef MPI
+      call bct2(tkea,device=.false.)
       !$acc data copy(tkea)
-      call bct2_GPU(tkea)
       call getcornert_GPU(tkea,nw1,nw2,ne1,ne2,sw1,sw2,se1,se2)
```

Comment:
Can we modify the `getcornert_GPU` subroutine as other **bcXXX** subroutine to take the `device` argument and simplify the code here?

Reply:
This will be done in a subsequent PR.  I just have not gotten to upgrading all subroutines in comm.F yet.

Reply:
Thanks for the clarification.